Sources/AppStoreServerLibrary/ChainVerifier.swift

```diff
@@ -132,7 +131,7 @@ class ChainVerifier {
         return await verifier.validate(leafCertificate: leaf, intermediates: intermediateStore)
     }
     
-    func getDate() -> Date {
+    nonisolated func getDate() -> Date {
         return Date()
     }
```

Comment:
Not sure what the intent on making a separate `getDate()` method was, but it must now be `nonisolated` so it can be called by the above method, which is also `nonisolated`.

Reply:
Note: I removed `getDate` since ChainVerifier is no longer a class that can be subclassed in tests (being an actor), opting for a parameter to related functions instead for mocking purposes. Please see the latest commit for this change.